src/core/geometry/qgsgeometry.h

```diff
@@ -1583,15 +1581,6 @@ class CORE_EXPORT QgsGeometry
 #endif
 ///@endcond
 
-    /**
-     * Returns the length of the QByteArray returned by asWkb()
-     *
-     * The optional \a flags argument specifies flags controlling WKB export behavior
-     *
-     * \since QGIS 3.16
-     */
-    int wkbSize( QgsAbstractGeometry::WkbFlags flags = QgsAbstractGeometry::WkbFlags() ) const;
```

Comment:
same as above:
Somebody else seems to have added QgsGeometry::wkbSize in the file meanwhile I was working on it as well. 
(according to https://github.com/qgis/QGIS/commit/7ac4b5188598890ea59951d19fbf02e2bb0738e0#diff-08789a83813c83fdba446c6ac938c962d1d26de41a1f235628e2caad0ede2762)
No idea what the correct code should be here. I have not been prompted for any merge suggestion during commit or push. Don't know if it's worth mentioning that maybe it got infiltrated when switching to my new personal remote repo.
Can anybody help me clarify whether the other change has correctly been included automatically or has it been included erroneously?